khal/khalendar/event.py

```diff
@@ -451,6 +458,8 @@ def relative_to(self, day, full=False):
                 body += ', ' + self.description.strip()
             if self.location.strip() != '':
                 body += ', ' + self.location.strip()
+            if self.categories.strip() != '':
+                body += ', ' + self.categories.strip()
```

Comment:
It's a matter of personal taste.

The right solution would be to have a configuration item for the order of fields. For example
> order_print_normal = Time, Title
> order_print_full = Title, Title, Description, Categories
> order_interactive = Title, Organizer, Location!, Date, Calendar, Categories
> order_editor = Title, Calendar, Location, Categories, Description, Date, Repeat, Alarm

With a !-mark if an item must be displayed even if it is empty. Would be useful to also mask "Alarm" in the editor, which I never use.